Modules/Microsoft365DSC/Dependencies/Manifest.psd1

```diff
@@ -83,6 +83,14 @@
         @{
             ModuleName      = 'ReverseDSC'
             RequiredVersion = '2.0.0.13'
+        },
+        @{
```

Comment:
Why do we have it listed twice, and can we please add it to the top of the dependency manifest? Also, I would prefer we don't ever call in the Az module. Everything we need should be available as part of the Microsoft.Graph cmdlets. E.g. Get-MgServicePrincipal instead of Get-AzADServicePrincipal.

Reply:
As Nik also mentions, this PR has the Az.Resources module mentioned twice. One can be removed.

I agree with Nik that it would be great to update to the Graph module, but that will require some more testing. Probably better to implement in a separate change.